app.js

```diff
@@ -85,17 +87,22 @@ const exampleEmbed = new Discord.MessageEmbed()
   )
   .setTimestamp()
   .setFooter("Happy Reading", "https://i.imgur.com/vugPtoT.png");
-  
+
 const rule = new schedule.RecurrenceRule();
 rule.dayOfWeek = [0, new schedule.Range(0, 6)];
 rule.hour = 10;
 rule.minute = 00;
 var startDay = 0;
 var endDay = 6;
 
-//creating a empty cron expression
-var cronExpression = `${rule.minute} ${rule.hour} * * ${startDay}-${endDay}`;
+const ruleForBMC = new schedule.RecurrenceRule();
+ruleForBMC.dayOfWeek = [0, new schedule.Range(0, 6, 2)];
+ruleForBMC.hour = 12;
```

Comment:
since these are same for every server, how about adding these to `.env`?

Reply:
Ok, I will try doing it